src/traces/histogram/attributes.js

```diff
@@ -71,6 +71,46 @@ module.exports = {
         ].join(' ')
     },
 
+    cumulative: {
```

Comment:
test image in https://github.com/plotly/plotly.js/pull/1189/commits/53b61aa796465534c46834a81e60f9ff19b2f781

One thing this showed is that we need a way to harmonize autobins across traces, and that it needs to know about `cumulative`. To make this example work I needed to manually extend the bin range for the smaller trace, otherwise its CDF ended too soon. Actually, CDFs *never* end, really... so perhaps the even better thing to do would be to look at the axis range and draw bins out to the edge. Anyway, fixing this will be a bigger project so I'll make an issue for it rather than try to address it here.